### PR TITLE
Add support for Confluent Platform 3.0.1.

### PR DESCRIPTION
**Pull request by dbtucker**
The confluent-kafka package was updated to reference the updated CP 3.0.1 tarball.   Icons were updated to core Kafka.

The other confluent-* packages were updated to reference the 3.0.1 docker images and leverage the improved naming service within the deployment (removing the need for explicit VIPs)

### CODE REVIEW

File: repo/packages/C/confluent-connect/2/resource.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "assets": {
+    "container": {
+      "docker": {
+        "image": "dbtucker/cp-kafka-connect"
```
Comment (gabrielhartmann) [inline repo/packages/C/confluent-connect/2/resource.json]: Did you want an explicit tag on this image?  This is going to default to `latest` I think.

File: repo/packages/C/confluent-kafka/8/package.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "packagingVersion": "3.0",
+  "name": "confluent-kafka",
+  "version": "1.1.12-3.0.2",
```
Comment (gabrielhartmann) [inline repo/packages/C/confluent-kafka/8/package.json]: This should be `1.1.12-3.0.1` correct?

File: repo/packages/C/confluent-schema-registry/2/marathon.json.mustache [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    "type": "DOCKER",
+    "docker": {
+      "image": "{{resource.assets.container.docker.image}}:{{registry.docker-tag}}",
+	  "forcePullImage": true,
+      "network": "BRIDGE",    
+      "portMappings": [ {
+          "containerPort": 8081,
+          "hostPort": 0,
+          "protocol": "tcp",
+    	  "labels": {
```
Comment (gabrielhartmann) [inline repo/packages/C/confluent-schema-registry/2/marathon.json.mustache]: There's a <tab> in the whitespace here.

Comment (dbtucker) [inline repo/packages/C/confluent-schema-registry/2/marathon.json.mustache]: Removed all tabs.

File: repo/packages/C/confluent-connect/2/resource.json (hunk 1 above)
Comment (dbtucker) [inline repo/packages/C/confluent-connect/2/resource.json]: For now, the tag is a separate value.   The image specification in marathon.json.mustache is built from this value and then the docker-tag.

Comment (keithchambers) [inline repo/packages/C/confluent-connect/2/resource.json]: I recommend sticking with an explicit version of the Docker image in the resource.json file vs. making it a variable.

There are a couple reasons why:
* DC/OS UI shows all the versions installed. This won't be accurate anymore.
* There is a tool that allows on-premise DC/OS users to create a local Universe mirror. We need to understand if there are implications if package assets become user configurable.

/cc @pyronicide for his thoughts.

Comment (pyronicide) [inline repo/packages/C/confluent-connect/2/resource.json]: +1 we really need the image to be a single variable (because of downstream tooling).

**Comment by dbtucker** [comment]
Per Keith's suggestion: removed docker-tag option so that images are now specified completely in resource.json.

**Pull request merged**